library_generation/DEVELOPMENT.md

```diff
@@ -0,0 +1,101 @@
+# Linting
+
+When contributing, ensure your changes to python code have a valid
+format.
+
+```
+python -m pip install black
+black library_generation
+```
+
+# Running the integration tests
+
+The integration tests build the docker image declared in
+`.cloudbuild/library_generation/library_generation.Dockerfile`, pull GAPIC
+repositories, generate the libraries and compares the results with the source
+code declared in a "golden branch" of the repo.
+
+It requires docker and python 3.x to be installed.
+
+```
+cd library_generation
```

Comment:
I think we can add a blurb in the start of the file to indicate that all scripts mentioned in this DEVELOPMENT.md is assumed to be performed under `library_generation` folder. At least for linting, unit tests, and integration tests.

Reply:
Ok, I added a blurb explaining this